models/mednist_gan/scripts/losses.py

```diff
@@ -0,0 +1,26 @@
+import torch
+
+disc_loss_criterion = torch.nn.BCELoss()
+gen_loss_criterion = torch.nn.BCELoss()
+real_label = 1
+fake_label = 0
+
+
+def prepare_batch(batchdata, device=None, non_blocking=False):
+    return batchdata["image"].to(device=device, non_blocking=non_blocking)
```

Comment:
We already have this function in: 
https://github.com/Project-MONAI/MONAI/blob/dev/monai/engines/utils.py#L130

Thanks.

Reply:
Yeah that works with all the keys changed to "reals".